### PR TITLE
12.0 account invoice date required

### PR DESCRIPTION
**Pull request by victor-champonnois**
New module that make the date_invoice field required in account.invoice_form and account.invoice_supplier_form.

### CODE REVIEW

File: account_invoice_date_required/i18n/fr_BE.po [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+#~ msgid ""
+#~ "The field Vendor is required, please complete it to validate the Vendor "
+#~ "Bill."
+#~ msgstr ""
+#~ "Le champ Vendeur est requis, veuillez le compléter pour valider cette "
+#~ "facture d'achat."
+
+#~ msgid "Validate an invoice with a negative total amount"
+#~ msgstr "Valider une facture avec un montant total négatif"
```
Comment (remytms) [inline account_invoice_date_required/i18n/fr_BE.po]: Not sure why this is commented.

File: account_invoice_date_required/i18n/fr_CH.po [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+#~ msgid ""
+#~ "The field Vendor is required, please complete it to validate the Vendor "
+#~ "Bill."
+#~ msgstr ""
+#~ "Le champ Vendeur est requis, veuillez le compléter pour valider cette "
+#~ "facture d'achat."
+
+#~ msgid "Validate an invoice with a negative total amount"
+#~ msgstr "Valider une facture avec un montant total négatif"
```
Comment (remytms) [inline account_invoice_date_required/i18n/fr_CH.po]: Not sure why this is commented. Maybe we can just remove the file and let weblate make the work ?

File: account_invoice_date_required/i18n/fr.po [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+#~ msgid ""
+#~ "The field Vendor is required, please complete it to validate the Vendor "
+#~ "Bill."
+#~ msgstr ""
+#~ "Le champ Vendeur est requis, veuillez le compléter pour valider cette "
+#~ "facture d'achat."
+
+#~ msgid "Validate an invoice with a negative total amount"
+#~ msgstr "Valider une facture avec un montant total négatif"
```
Comment (remytms) [inline account_invoice_date_required/i18n/fr.po]: Not sure why this is commented.

Comment (victor-champonnois) [reply account_invoice_date_required/i18n/fr.po]: removed

File: account_invoice_date_required/i18n/fr_CH.po (hunk 2 above)
Comment (victor-champonnois) [reply account_invoice_date_required/i18n/fr_CH.po]: OK, I removed the po files

File: account_invoice_date_required/i18n/fr_BE.po (hunk 1 above)
Comment (victor-champonnois) [reply account_invoice_date_required/i18n/fr_BE.po]: removed

**Review by carmenbianca — APPROVED** [review]
LGTM

**Pull request closed without merging**